validator/schema-v1.json

```diff
@@ -75,34 +75,34 @@
                 "unit": {
                   "type": "string",
                   "enum": [
-                    "Count",
-                    "Percent",
-                    "Ratio",
-                    "Seconds",
-                    "Centiseconds",
-                    "Milliseconds",
-                    "Microseconds",
-                    "Nanoseconds",
-                    "Bytes",
-                    "Kilobytes",
-                    "KilobytesPerSecond",
-                    "Megabytes",
-                    "BytesPerSecond",
-                    "BitsPerSecond",
-                    "MegabytesPerSecond",
-                    "MegabytesPerMillisecond",
-                    "CountPerSecond",
-                    "CountPerMinute",
-                    "GigabytesPerSecond",
-                    "Gibibytes",
-                    "Mebibytes",
-                    "dBm",
-                    "Gigabytes",
-                    "Terabytes",
-                    "Megahertz",
-                    "Watt",
-                    "Joule",
-                    "Other"
+                    "count",
```

Comment:
just using lowercase I think fails with the Cloud integrations files

Reply:
yes it's true it is modified when the automations generate the specs . I will change this and do it the same way